src/awesome_map/CustomSwipeGesture.js

```diff
@@ -83,6 +83,23 @@ define(function(require) {
                     inst.trigger(this.name + ev.direction, ev);
                 }
             }
+        },
+
+        /**
+         * This is a custom function, not a standard method on Hammer gestures.
+         * It will replace the existing swipe gesture Hammer uses with this one.
+         */
+        register: function() {
+            // Install into the detection stack, replacing the default gesture.
+            var gestures = Hammer.detection.gestures;
+            for (var i = 0, n = gestures.length; i < n; i++) {
+                if (gestures[i].name === CustomSwipeGesture.name) {
```

Comment:
could/should ```this``` be used instead?  Seems like it would be more consistent with the rest of the file using ```this```